joomla/filesystem/CVE-2022-23794.yaml

```diff
@@ -0,0 +1,9 @@
+
+title:     Path Disclosure within joomla/filesystem class
+link:      https://developer.joomla.org/security-centre/871-20220302-core-path-disclosure-within-filesystem-error-messages.html
+cve:       CVE-2022-23794
+branches:
+    master:
+        time:     2022-03-29 18:00:00
+        versions: ['<1.6.2', '<2.0.1']
```

Comment:
this is the wrong way to specify that it is patched in 2 different versions. You need 2 branches for that, to be able to mark `<1.6.2` and `>=2.0.0, <2.0.1` as affected.

Reply:
Ah ok Nowdays we have 2.x-dev and 1.x-dev and no master any more. Will take a look and update the PR later. Thanks!